app/experimenter/nimbus-ui/src/index.test.tsx

```diff
@@ -0,0 +1,20 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+ import React from 'react';
```

Comment:
Wayward space?  Shouldn't autoformat catch this?

Reply:
idk if autoformat will catch it but it's a byproduct of L3's single space and VS Code trying to match indentation. We have grumbled about it in FXA repeatedly 😛

Reply:
What's L3?

Reply:
Line 3 😄

Reply:
Ugh yeah, for some reason VS Code does a weird extra space when we paste the license at the top. I'd think Prettier would fix it (does it in FxA? I tend to catch it and fix it manually), I'll get that working next to see.

Reply:
IIRC I don't think Prettier fixes it in FxA. One solution is to not use `/**` format and instead use `//` format for these license headers, though.